courses/fundamentals_of_ada/adv_065_primitives.rst

```diff
@@ -83,11 +91,153 @@ Implicit Primitive Operations
 
       package P is
          type T1 is range 1 .. 10;
-         -- implicit: function "+" (Left, Right : T1) return T1;
+         -- implicit
+         -- function "+" (Left, Right : T1) return T1;
       end P;
       ...
       procedure Main is
          V1, V2 : T1;
       begin
          V1 := V1 + V2;
       end Main;
+
+---------------------------
+Recap. on type derivation
```

Comment:
I don't think "Recap" is an abbreviation (so no ".") but maybe "Review" would be better understood by non-English speakers

Reply:
Got it, interesting, recap does indeed exist, unlike French were we only have the longer-form of recapitulate